Configure

```diff
@@ -551,6 +539,14 @@ my @seed_sources = ();
 while (@argvcopy)
 	{
 	$_ = shift @argvcopy;
+
+	# Support env variable assignments among the options
+	if (m|^(\w+)=(.+)?$|)
```

Comment:
Someone will want specify something with spaces, in which case there probably should be a way to specify that...

Reply:
Do you know what people will complain about? "Why can't I specify CLFAGS? That's how we do it elsewhere..." Well, CFLAGS environment variable is ignored now, answer is pass on command line, say with `./config $CFLAGS`. But question here is if one decides to do something, would it be appropriate to do something different. BTW, CFLAGS might be too loaded to think about, consider rather LDFLAGS as think experiment...

Reply:
I'm not sure what you mean with "specify something with spaces"...  wouldn't something like this suffice?

```
./config SPACE=' '
```

Yeah, I know about the CFLAGS / LDFLAGS demand, and am currently having another look.  It should be fairly easy if we do it the same way as the GNU folks, i.e. support CFLAGS, CPPFLAGS and LDFLAGS.

Reply:
> I'm not sure what you mean ...

Environment variable with space in its value. `SPACE=' '` is kind of bad example, because it kind of works out now. But try `SPACE='a b'`...

Reply:
Of course there is difference between passing SPACE='a b' to ./config and ./Configure... If passed to ./Configure it works as expected. So there was a bit of confusion there, sorry...

Reply:
Ahhhhhh, `config` doesn't quotify the arguments to `Configure`.... ok, that should be easy enough to fix, I hope

Reply:
Why `(.+)?` and not `(.*)`?

Reply:
Because I want an empty value (no characters at all) to generate `undef`

Reply:
`config` to `Configure` argument passing fixed, or so I hope